templates/index.json

```diff
@@ -192,9 +202,9 @@
         "background_style": "none",
         "button_label": "",
         "button_link": "",
-        "swipe_on_mobile": false,
         "color_scheme": "background-1",
         "columns_mobile": "1",
+        "swipe_on_mobile": false,
         "padding_top": 36,
         "padding_bottom": 36
       }
```

Comment:
Hey @allylane, when you create a theme that's linked to your PR, you want to enable one way sync so that any edits made in the theme editor while testing your work isn't updated back to the PR. 
So here for example, there shouldn't be any change to this file 🤔 
![](https://screenshot.click/14-40-da2k8-xzzf8.png)

Reply:
Yea, whoops 🤦‍♂️